#154715726 User should be effectively onboarded

sannytee opened the pull request:
#### What does this PR do?
Enables easy navigation of the page and onboard user

#### Description of Task to be completed?
>- redesign the homepage template


#### How should this be manually tested?
* from terminal, run: `git clone https://github.com/andela/kenya-rc.git` to clone repository
* cd into the cloned repository and run: `meteor npm install` to install all required dependencies
* run `reaction` to startup application and open browser at localhost:3000. 


#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
154715726

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-02-06 at 11 07 46 am" src="https://user-images.githubusercontent.com/28694656/35854391-82ad8dfe-0b30-11e8-9939-83c65591c072.png">
<img width="1437" alt="screen shot 2018-02-06 at 11 08 30 am" src="https://user-images.githubusercontent.com/28694656/35854390-828cd83e-0b30-11e8-864d-3efd5f352698.png">
<img width="1428" alt="screen shot 2018-02-06 at 11 08 37 am" src="https://user-images.githubusercontent.com/28694656/35854386-81e6ffcc-0b30-11e8-8e31-61aae8608f4b.png">
<img width="1433" alt="screen shot 2018-02-06 at 11 08 50 am" src="https://user-images.githubusercontent.com/28694656/35854385-81c32b7e-0b30-11e8-8204-dfef63f1bf5e.png">
<img width="1440" alt="screen shot 2018-02-06 at 11 08 59 am" src="https://user-images.githubusercontent.com/28694656/35854383-81a1ede2-0b30-11e8-98f6-5820b8046c3f.png">



#### Questions:
N/A

sannytee committed:
feature(onboarding): User should be effectively onboarded

- add images to the resources folder

- create less file for the homepage templates

- create the homepage templates

- bind the template to the application

michaelogala commented on client/templates/homepage.html:
```diff
@@ -0,0 +1,21 @@
+    {{#if hasAdminAccess}}
+    <div>
+      {{> products}}
+    </div>gt {{else}}
```
Shouldn't the `{{else}}` be on its own line?

michaelogala commented on imports/plugins/included/default-theme/client/styles/homepage.less:
```diff
@@ -0,0 +1,157 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:100,300i,400,700'); 
+ 
+.fashionStatement { 
```
Stick to separating class names with hyphen => `.fashion-statement`

michaelogala commented on imports/plugins/included/default-theme/client/styles/homepage.less:
```diff
@@ -0,0 +1,157 @@
+  height: 100%; 
+  width: 100%; 
+  background: black; 
+  opacity: 0.2 !important; 
```
We will not be using `!important` to style any element in this project. Find another way to make this style work.

michaelogala commented on imports/plugins/included/default-theme/client/styles/homepage.less:
```diff
@@ -0,0 +1,157 @@
+ 
+ 
+.about-text { 
+  padding-top: 50px !important; 
```
Please remove all occurrences of `!important` from this PR

michaelogala commented on imports/plugins/included/ui-home/client/templates/about.html:
```diff
@@ -0,0 +1,42 @@
+        <div>
+          <h2 class="text-center about-text">ORIGINS</h2>
+    
+          <p style="text-align: center; line-height: 2.5em; width: 50%; margin: 0 auto;  font-family: 'Roboto', sans-serif; font-weight:100; font-size: 17px;">
```
Why are we using inline style here. Can the styling not be placed in an appropriate `.less` file?

michaelogala reviewed (changes requested):
Please respond to the comments I've made

sannytee committed:
chore: refactor-code

-Remove inline styles

- change naming convention

- remove `!Important` attribute

michaelogala reviewed (approved):
👍
